backend/question-service/server/app/routers/admin.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import HTTPException, Query, APIRouter
+from typing import Optional
+import sys
+
+sys.path.append("..")
+from core.database import collection
+from utils.jsend import JSendStatus, jsend_response
+
+
+admin_router = APIRouter(
+    prefix="/admin",
+    tags=["Admin"],
+)
+
+
+@admin_router.post("/questions")
+async def create_question(question: dict):
```

Comment:
Is there a way to check if the question body is valid i.e. required/optional fields, etc.?

Reply:
At the moment no. I assume we validate through the frontend first, but I'll explore adding a model for the question data soon in the future. Hope this is okay